result/flow.yaml

```diff
@@ -116,28 +135,49 @@ components:
           description: >-
             The current rate of valid frames received
           type: number
-        min_latency_ns:
+        loss:
           description: >-
-            The minimum latency in nanoseconds for valid frames received
+            The percentage of lost frames
           type: number
-        max_latency_ns:
-          description: >-
-            The maximum latency in nanoseconds for valid frames received
+        timestamps:
+          $ref: '#/components/schemas/Metric.Timestamp'
+        latency:
+          $ref: '#/components/schemas/Metric.Latency'
+
+    Metric.Timestamp:
+      description: |-
+        The container for timestamp metrics.
+      type: object
+      properties:
+        first_timestamp_us:
```

Comment:
Based on our previous discussion, all timestamps can be `nano`

Reply:
d9ba3bb